NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Fw ticket35 view reactions

Description
Added View Reactions function in Post Details to view reactions for a post. [Trello Ticket #35](https://trello.com/c/VDjvONZs/35-35-view-reactions-on-a-post)

Type of Change
-Added new, non-breaking feature.

Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout fw-ticket35-viewReactions
5. If you don't see the most recent version of my files, git pull origin fw-ticket35-viewReactions
6.At the moment, Reaction table is empty so insert some values into it
 INSERT INTO Reaction (Id, ReactionName, ImageUrl) VALUES
(1, 'Like', 'https://cdn-icons-png.flaticon.com/512/25/25297.png'),
(2, 'Dislike', 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ6Jt4fWTTKq-a3g4LAk1FNBRURO87dt5UDjg&s');

7. At the moment, PostReaction table is empty so insert some values into it for Post with Id=1 to have some reactions:                           
INSERT INTO PostReaction (PostId, ReactionId, UserProfileId)
VALUES 
    (1, 1, 1),
    (1, 2, 2),
    (1, 1, 3);

8. Check that PostReaction is populated: select * from PostReaction
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in your client folder. Run "npm start" in your terminal.
11. Login as an admin. foo@bar.com for the email parameter. Leave the password text box blank. Click "Login".
12. Click on Posts at the top of the navbar.
13. Once on the posts page, it should list ALL posts as normal. Click the post, titled “[morph front-end markets](http://localhost:3000/posts/1)”(PostId=1). You should see thumbs up and thumbs down with count next to them.

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
